build.gradle

```diff
@@ -1,40 +1,48 @@
 buildscript {
     repositories {
+        // These repositories are only for Gradle plugins, put any other repositories in the repository block further below
         maven { url = 'https://maven.minecraftforge.net' }
```

Comment:
Be aware that we are moving to NeoForge which may work differently

Reply:
Yeah there hasn't been a lot of documentation out for them yet, but I'll adjust it when there is